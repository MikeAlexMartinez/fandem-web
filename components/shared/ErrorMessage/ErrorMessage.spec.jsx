import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import ErrorMessageComponent from './ErrorMessage';

describe('<ErrorMessage />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(
      <ErrorMessageComponent
        message="Test Message"
      />,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
