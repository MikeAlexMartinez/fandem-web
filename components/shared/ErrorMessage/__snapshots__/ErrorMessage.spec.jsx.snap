// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> renders and matches the snapshot 1`] = `
<ErrorMessage
  classes={
    Object {
      "columnItem": "ErrorMessage-columnItem-2",
      "container": "ErrorMessage-container-1",
      "icon": "ErrorMessage-icon-3",
    }
  }
  message="Test Message"
/>
`;
